src/plugins/optimize-graph-plugin.js

```diff
@@ -479,7 +479,7 @@ function constructAssetToChunkMap(bundle) {
  * @param {(fn: string, url: string, quote: string) => string | null | undefined} replacer Return replacement code, or `null`/`undefined` to preserve the matched call.
  */
 function replaceSimpleFunctionCall(code, replacer) {
-	return code.replace(/([a-z$_][a-z0-9$_]*)\((['"`])(.*?)\2\)/gi, (s, fn, quote, url) => {
+	return code.replace(/([a-z$_][a-z0-9$_]*)\((['"`])((?:(?!\2)[^\\]|\\.)*?)\2\)/gi, (s, fn, quote, url) => {
```

Comment:
While we're fixing this, I'd also like to add a negative lookbehind that ensure it doesn't match dotted identifiers like `document.createElement(`:

```suggestion
	return code.replace(/(?<![.\w])([a-z$][\w$]*)\((['"`])((?:(?!\2)[^\\]|\\.)*?)\2\)/gi, (s, fn, quote, url) => {
```

Reply:
Ah of course, I'll add that - I assume the `a-z$` should also include `_` since that's a valid starting char for an identifier

Reply:
Hah - yes. Good catch.